### PR TITLE
Prcda uiho

### PR DESCRIPTION
**Pull request by howew**
This branch has the changes to the PRCDA and UIHO algorithms for this year's CFD.

### CODE REVIEW

File: src/main/resources/prcdauiho/prcda.csv [hunk 1]
```diff
@@ -0,0 +1,506 @@
+﻿state,county,prcda2017,prcda2020
```
Comment (depryf) [inline src/main/resources/prcdauiho/prcda.csv:1]: You renamed all the packaged, but left this one as the original "prcdauiho"; I think it needs to be "prcda" to be consistent with everything else you did.

File: src/test/java/com/imsweb/algorithms/prcda/PrcdaUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        states.addAll(invalStates);
+        states.addAll(ENTIRE_STATE_PRCDA);
+
+        List<String> counties = new ArrayList<>();
+        counties.addAll(validCounty);
+        counties.addAll(invalCounty);
+
+        PrcdaInputDto input = new PrcdaInputDto();
+
+        for (String state : states) {
```
Comment (depryf) [inline src/test/java/com/imsweb/algorithms/prcda/PrcdaUtilsTest.java:54]: It is not good practice to unit test to exhaustively all the possible combinations.

Unit tests are meant to test execution path and find bugs in the code. From that point-of-view, a test should not go multiple times through the same execution path. 

This feels more like a way to validate your data, but that's really not the purpose of a unit test.

If you want to to run this as a one-time process to validate your data, that's fine. But for the test that runs moving forward, it would be better to only test a few cases: a normal one and then some corner cases related to unknown. 

Assuming this test is reasonable fast (I am sure it is), you can leave this as-is. But please keep this in mind for new tests that your write in the future.

**Review by depryf — CHANGES_REQUESTED** [review]

**Comment by depryf** [comment]
@howew can you also please add a changelog entry in the changelog file? Thanks.

File: src/main/resources/prcdauiho/prcda.csv (hunk 1 above)
Comment (howew) [reply src/main/resources/prcdauiho/prcda.csv:1]: Yup.  Good catch!  Will fix it ASAP.

File: src/test/java/com/imsweb/algorithms/prcda/PrcdaUtilsTest.java (hunk 2 above)
Comment (howew) [reply src/test/java/com/imsweb/algorithms/prcda/PrcdaUtilsTest.java:54]: Noted!  Thank you.

**Comment by howew** [comment]
I've made the changes.  Thanks.

**Review by depryf — APPROVED** [review]
